packages/flame/lib/src/components/time_track_component.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:async';
```

Comment:
Can you create a subdirectory inside `component` called `debug` too where these can live? And eventually we can move over other ones too, like the ones tracking fps (probably add those to the debug barrel file too).